### ISSUE TITLE
A few privacy-related requests

### ISSUE THREAD

**Issue by lifepillar** [doug-leith/appFirewall#9]
This is some information that could at least be clarified in the Readme, if no alternative is possible:

1. The “security settings” that are uploaded the first time appFirewall is launched include the list of installed apps (and their associated xattr attributes). Such a list may be a unique or almost unique fingerprint in many cases (similar to font fingerprinting in browsers). Is uploading that information necessary?

2. The Readme mentions that appFirewall “periodically uploads a sample of the connections” of a randomly selected app. Is that done transparently, or is the user notified before such an upload takes place? Also, how often does that happen? Ideally, I would like to be able to review any information that the app would like to upload *before* an upload takes place. This would mitigate the concern, correctly pointed out in the Readme, that some apps might leak sensitive data.

3. The Readme also mentions that on first launch, a helper tool is installed to gain the necessary privileges. How is it named? Where is it installed? Is it removed by dragging and dropping the app into the Trash? More generally, it would be nice if the Readme included the list of paths for the complete removal of the app (e.g., `~/Library/Application Support/appFirewall`, `~/Library/Preferences/com.leith.appFirewall.plist`—anything else?).

Keep up with your great work!

**Comment by lifepillar** [comment]
>a helper tool is installed

Ok, I've found this myself. It's inside tha app bundle, so deleted when the app is deleted.

**Comment by lifepillar** [comment]
No, scrap that. It's a launchd daemon, which keeps running when appFirewall has quit. I think that it should be mentioned in the Readme: to uninstall appFirewall it is not enough to trash the app, you also need to delete the launchd plist (and kill the helper process or restart, if it is running).

**Comment by jameskingston** [comment]
It also installs /Library/PrivilegedHelperTools/com.leith.appFirewall-Helper
